server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -1007,7 +1007,12 @@ public PartialPath getBelongedStorageGroup(PartialPath path) throws StorageGroup
 
   /** Get all storage group paths */
   public List<PartialPath> getAllStorageGroupPaths() {
-    return mtree.getAllStorageGroupPaths();
+    try {
+      return mtree.getAllStorageGroupPaths();
+    } catch (MetadataException e) {
+      logger.warn("Something wrong when collecting storage groups");
```

Comment:
This should be an error rather than warn.

Reply:
Fixed.